var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -756,30 +754,29 @@ def configure_args(self):
 
         # CUDA support
         # See https://www.open-mpi.org/faq/?category=buildcuda
-        if spec.satisfies('@1.7:'):
-            if '+cuda' in spec:
-                # OpenMPI dynamically loads libcuda.so, requires dlopen
-                config_args.append('--enable-dlopen')
-                # Searches for header files in DIR/include
-                config_args.append('--with-cuda={0}'.format(
-                    spec['cuda'].prefix))
-                if spec.satisfies('@1.7:1.7.2'):
-                    # This option was removed from later versions
-                    config_args.append('--with-cuda-libdir={0}'.format(
-                        spec['cuda'].libs.directories[0]))
-                if spec.satisfies('@1.7.2'):
-                    # There was a bug in 1.7.2 when --enable-static is used
-                    config_args.append('--enable-mca-no-build=pml-bfo')
-                if spec.satisfies('%pgi^cuda@7.0:7'):
-                    # OpenMPI has problems with CUDA 7 and PGI
-                    config_args.append(
-                        '--with-wrapper-cflags=-D__LP64__ -ta:tesla')
-                    if spec.satisfies('%pgi@:15.8'):
-                        # With PGI 15.9 and later compilers, the
-                        # CFLAGS=-D__LP64__ is no longer needed.
-                        config_args.append('CFLAGS=-D__LP64__')
-            else:
-                config_args.append('--without-cuda')
+        if '+cuda' in spec:
```

Comment:
simplified this cause there's a conflict above `conflicts('+cuda', when='@:1.6')`